tests/Doctrine/Tests/DBAL/Driver/AbstractSQLServerDriverTest.php

```diff
@@ -52,6 +52,8 @@ protected function getDatabasePlatformsForVersions()
             array('11.00.2100.99', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
             array('11.00.2101', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
             array('12', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
+            array('14', 'Doctrine\DBAL\Platforms\SQLServer2014Platform'),
```

Comment:
Please revert the changes here, the platforms do not exist yet. Also this should not be necessary for this PR. Do you have any problems with that?